src/sortable.js

```diff
@@ -20,119 +20,114 @@ angular.module('ui.sortable', [])
           uiSortable: '='
         },
         link: function(scope, element, attrs, ngModel) {
+          'use strict';
```

Comment:
I totally in favor of 'use strict', but this might cause errors in some ASP.Net environments.
jQuery prior v3 also didn't use 'use strict' for exactly that reason.